.github/workflows/ci.yml

```
@@ -70,30 +75,134 @@ jobs:
+          fi
+
+      - name: Reject #[allow] / @ts-ignore / oxlint-disable
+        run: |
+          set +e
+          # Multi-line aware Rust scanner: `git grep` is line-oriented, so
+          # `#[allow(\n    dead_code,\n)]` slips through. Slurp each .rs file
+          # and walk every `#[allow(...)]` group with a regex that spans
+          # newlines, then match the forbidden tokens inside the captured
+          # argument list.
+          BAD_RUST=$(python3 - <<'PY'
+import re, subprocess, sys
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)